taskcluster/ac_taskgraph/target_tasks.py

```diff
@@ -74,10 +75,28 @@ def filter(task):
 
 
 def get_gv_version(repo, revision):
+    gecko_kt_path = get_gecko_kt_path(repo, revision)
     gecko_kt = repo.run(
-        "show", f"{revision}:android-components/plugins/dependencies/src/main/java/Gecko.kt"
+        "show", f"{revision}:{gecko_kt_path}"
```

Comment:
We could, although any error raised by `git` will be caught under the generic `CalledProcessError`. So we would try to fallback to the old path even though the first command failed for an unrelated reason. 

In the current implementation, we're safer because `git merge-base --is-ancestor` fails by design.

Reply:
I'm happy to make any follow-up changes, if you wish 🙂